### DISCUSSION TITLE
Awesome!

### DISCUSSION THREAD

**Discussion by gitinthebus** [Stirling-Tools/Stirling-PDF#56] [General]
This is one of the most useful applications I've seen in a long time. Dockerized and with a simple web UI.
Thanks!

**Comment by Frooodle** [comment]
Thanks! Appreciate the complements and download 
Hoping to keep adding more features every week, so the application should grow even more as time goes

**Comment by YoloTerry** [comment]
How to extra the table and remain original format from pdf created by scanning ?

**Comment by bryce-carson** [comment]
It is a very good application! Adobe doesn't have a stick to shake at the performance and usability of this, especially for individual users! I'll contribute or fund if I can. :)

**Comment by ovidius1980** [comment]
I love it. Just a question. Can I have the old interface with V2?
